public/icon/filter.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
in the latest I have added this to the icon font and to the drawer as an <Icon> component